src/visualizers/Turtle.ts

```diff
@@ -84,15 +74,33 @@ positionally to the domain elements. Range and domain must be the same length.
 - strokeColor: The color used for drawing the path.
      **/
     strokeColor: {
-        default: '#ff0000',
+        default: '#c98787',
         type: ParamType.COLOR,
```

Comment:
I am getting this new color scheme as pink on brick red -- is that what you'd like for the default turtle visualization? (I totally defer to your design judgment, just checking/verifying that's what you intend.)

Reply:
Roger that. My taste in colors is terrible, so let's go with yours. :) Resolving.